ignore_words.txt

```diff
@@ -4,3 +4,6 @@ WAN
 filname
 ans
 entrie
+dout
+mane
```

Comment:
Hi @PipKat, I think this is here because these typos come from code generated automatically by the Server, hence this should be fixed at the server level. I'll start an issue for the removal of these exceptions with dedicated PRs server-side. Usually however this is kept because sadly correcting them would involve breaking-changes. We just have to find the right time to introduce them.